images/runtime/dotnetcore/3.0/Dockerfile.base

```diff
@@ -0,0 +1,57 @@
+# dotnet tools are currently available as part of SDK so we need to create them in an sdk image
+# and copy them to our final runtime image
+FROM mcr.microsoft.com/dotnet/core/sdk:3.0.100 AS tools-install
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-sos
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-trace
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-dump
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-counters
+
+FROM debian:buster-slim
```

Comment:
Note that we are NOT  using `oryx-run-base` here since it is based on `debian:stretch`